plugin-gestures/src/main/java/com/mapbox/maps/plugin/gestures/GestureState.kt

```diff
@@ -0,0 +1,56 @@
+package com.mapbox.maps.plugin.gestures
+
+import com.mapbox.android.gestures.AndroidGesturesManager
+
+internal class GestureState(private val gesturesManager: AndroidGesturesManager) {
+  enum class Type {
+    DoubleTap,
+    Scale,
+    ScaleQuickZoom,
+    Shove,
+  }
+
+  private val rememberedGestureSettings = mutableMapOf<Type, Boolean>()
```

Comment:
The meaning of the boolean doesn't seem to have clear relationship to the type.

Reply:
Renaming to `savedGestureIsEnabledMap`